components/user/UserLink.js

```diff
@@ -2,6 +2,17 @@ import colors from "@config/icons.json";
 import getIcon from "@components/Icon";
 import Link from "@components/Link";
 import Edit from "@components/account/manage/Edit";
+import styles from '../../styles/animation.module.css';
```

Comment:
Does it need `.module.` in the name? The other 2 files in the folder don't have that

Reply:
`.module.` added because by default NextJS supports direct `.css ` extension for global styles to use it we have to import that in app `_app.js` file if we don't do that it shows this error at Compile Time. To create local scope css styles we have to use `.module.` extension.  for more information here the documentation for NextJS CSS module https://nextjs.org/docs/pages/building-your-application/styling/css-modules
![1](https://github.com/EddieHubCommunity/LinkFree/assets/139515239/ea9fbe4a-c814-43fe-94ce-02541b556d7c)

Reply:
ok thank you 👍